### PR TITLE
Get url

### PR DESCRIPTION
**Pull request by ravescovi**

### CODE REVIEW

File: gladier/client.py [hunk 1]
```diff
@@ -630,6 +639,7 @@ def run_flow(self, flow_input=None, use_defaults=True):
                  f'"{flow["action_id"]}"')
         if flow['status'] == 'FAILED':
             raise gladier.exc.ConfigException(f'Flow Failed: {flow["details"]["description"]}')
+        flow['run_url'] = 'https://app.globus.org/flows/%s/runs/%s' % (flow_id,flow["action_id"])
```
Comment (NickolausDS) [inline gladier/client.py:642]: It's a bad idea to modify the original automate response. If we want `run_url` to be present, we should ask the Automate folks to add it.

**Comment by ravescovi** [comment]
Why so? I though about leaving it here in the meantime.
Is there a way of returning it in a recoverable way?

**Comment by NickolausDS** [comment]
It causes maintainability problems down the road. The problem goes like this: 

1. We change the automate response to include `run_url`. 
2. Our users start using the `run_url` attribute
3. The automate team adds a new `run_url` feature, perhaps using the same attribute or a different one
4. Gladier now overwrites automate's `run_url` or there are two in the response. We can't simply remove it because our users depend on the older version.

`run_url` itself is probably less likely to cause problems, but generally that's the problem with tinkering with responses. We could add `get_run_url()` that does the same thing.

**Comment by ravescovi** [comment]
Good idea. I will change it.

**Comment by NickolausDS** [comment]
Can you remove the other code?

File: gladier/client.py [hunk 2]
```diff
@@ -310,6 +311,14 @@ def get_flow_definition(self):
...
                                           '"gladier.GladierBaseTool"')
 
+    def get_flow_steps(self):
+        """
+        Get the flow steps in an ordered list attached to this class.
+
+        :return: A list of Step names
+        """
+        return flow_generation.get_ordered_flow_states(self.get_flow_definition(self))
+
```
Comment (NickolausDS) [inline gladier/client.py:321]: I'm still seeing the code for `get_flow_steps` here.

**Comment by ravescovi** [comment]
That should be fixed now.

**Pull request merged**